front/app/modules/commercial/insights/admin/containers/Insights/Details/Network/index.tsx

```diff
@@ -181,37 +192,64 @@ const Network = ({ params: { viewId } }: WithRouterProps) => {
   const onZoomEnd = ({ k }: { k: number }) => setZoomLevel(k);
 
   const onZoomIn = () => {
-    forceRef.current?.zoom(zoomLevel + zoomStep);
+    networkRef.current?.zoom(zoomLevel + zoomStep);
   };
 
   const onZoomOut = () => {
     zoomLevel - zoomStep > zoomStep
-      ? forceRef.current?.zoom(zoomLevel - zoomStep)
-      : forceRef.current?.zoom(zoomStep);
+      ? networkRef.current?.zoom(zoomLevel - zoomStep)
+      : networkRef.current?.zoom(zoomStep);
   };
 
   const nodeColor = (node: Node) =>
     nodeColors[node.color_index % nodeColors.length];
 
+  const exportNetwork = () => {
+    const srcCanvas = document.getElementsByTagName('canvas')[0];
```

Comment:
It's a bit of a hack but exporting canvas in this way means you don't have a background color. This results in very dark grey background for the exported image. So what we do here is we create a second canvas, paint it white and put the original canvas on top to ensure a white background.
https://stackoverflow.com/questions/18609715/html5-canvas-todataurl-image-has-no-background-color

Reply:
That's ok, maybe the comment could be more explicit, also a bit puzzled with the double width and height (is it only to get a bigger image than what's being rendered ? how come copying the canavas image scales correctly ?), but it all works well so we're good !

Reply:
Yes, good point on the comment, I added a bit more info there. 
On the *2 -I am also not 100% sure why it's needed, I suspect it has to do with some internal scaling of our viz library but I tested in cross-browser and it seems to look ok everywhere 🤷‍♀️